src/prospectScraper/ProspectScraper.cs

```diff
@@ -0,0 +1,636 @@
+﻿using CsvHelper;
+using HtmlAgilityPack;
+using prospectScraper.DTOs;
+using prospectScraper.Maps;
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+
+namespace prospectScraper
+{
+    public class ProspectScraper
+    {
+        public void RunTheBigBoards(bool parseDate = true)
+        {
+            File.WriteAllText($"logs{Path.DirectorySeparatorChar}Status.log", "");
+            File.WriteAllText($"logs{Path.DirectorySeparatorChar}Prospects.log", "");
+
+            Console.WriteLine("Getting data...");
+
+            var webGet = new HtmlWeb
+            {
+                UserAgent = "Mozilla/5.0 (Windows NT 6.3; WOW64; rv:31.0) Gecko/20100101 Firefox/31.0"
+            };
+            var document1 = webGet.Load("https://www.drafttek.com/2021-NFL-Draft-Big-Board/Top-NFL-Draft-Prospects-2021-Page-1.asp");
+            var document2 = webGet.Load("https://www.drafttek.com/2021-NFL-Draft-Big-Board/Top-NFL-Draft-Prospects-2021-Page-2.asp");
+            var document3 = webGet.Load("https://www.drafttek.com/2021-NFL-Draft-Big-Board/Top-NFL-Draft-Prospects-2021-Page-3.asp");
+            var document4 = webGet.Load("https://www.drafttek.com/2021-NFL-Draft-Big-Board/Top-NFL-Draft-Prospects-2021-Page-4.asp");
+            var document5 = webGet.Load("https://www.drafttek.com/2021-NFL-Draft-Big-Board/Top-NFL-Draft-Prospects-2021-Page-5.asp");
```

Comment:
This is the one that I will eventually get to. We can eventually use polymorphism to allow any sort of draft, and we can find a way to make this scale based on that draft type. Prefer the MLB? NHL? NBA? Well, maybe, just maybe someday, that will be possible.

Reply:
Check out #25 as there is some good stuff there, but I haven't merged it yet because we're still moving stuff around all over the place.

Reply:
Awesome! I'll stop moving things for now so we can get some of this in then. 👍